mergin/client.py

```diff
@@ -52,6 +52,8 @@ class ClientError(Exception):
 class SyncError(Exception):
     pass
 
+class UploadError(Exception):
```

Comment:
not sure if we need new error here, we use SyncError for both pull and push issues